src/modules/Humm90PlatinumCardExisting.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="cards">
+    <CardsList>
+      <slot />
+    </CardsList>
+  </div>
+  <p class="mc__title">Have a humm®90 Platinum Mastercard®?</p>
+  <p class="mc__subtitle">Enjoy interest-free shopping now!</p>
+  <ol class="mc__list">
+    <li>Add the item and continue to checkout</li>
+    <li>Choose interest-free as your payment option</li>
+    <li>Enter your card details and choose a plan listed below:</li>
+  </ol>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import CardsList from 'src/modules/CardsList.vue'
+
+export default defineComponent({
+  name: 'Humm90PlatinumCardExisting',
+  components: {
+    CardsList,
+  },
+})
+</script>
+
+<style scoped lang="scss">
+@import 'src/styles/cards.scss';
```

Comment:
This looks identical to the mastercard existing component – is that right? If they're identical, maybe you should make a reusable component to use for both instead of multiple separate ones?

Reply:
oof. nice one!
Just created ExistCard and ApplyCard to be used on both QMC and Humm90

Reply:
Perfect!